itail.el

```diff
@@ -166,7 +166,8 @@ clearing and filtering
 Very useful when the tail has had a great deal of information dumped
 to it and emacs can not keep up"
   (interactive)
-  (insert (concat "reloading " (buffer-name (current-buffer))))
+  (let ((inhibit-read-only t))
+    (insert (concat "reloading " (buffer-name (current-buffer)))))
```

Comment:
By the way, the `insert` here and the one below should maybe have a newline after it